incubator-tools/advance_table_line_enhancement/line_enhancement_basic_flow.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>How is this different from the previous fp_tables_to_csv ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
In the line enhancement basic flow, we enhance the table lines and then parse them through the Form Parser. On the other hand, in the fp_tables_to_csv process, we directly convert the Form Parser's table output into a CSV file.

Reply:
Ok, can you add that to the description?

Reply:
Updated